openstack/rds/v3/backups/requests.go

```diff
@@ -170,14 +170,37 @@ type RestoreToNewOpts struct {
 	RestorePoint     RestorePoint              `json:"restore_point" required:"true"`
 }
 
+type RestorePITROpts struct {
+	Source *Source `json:"source"`
+	Target *Target `json:"target"`
+}
+
+type Source struct {
+	InstanceId   string `json:"instance_id" required:"true"`
+	RestoreTime  int64  `json:"restore_time" required:"true"`
+	Type         string `json:"type" required:"true"`
+}
+
+type Target struct {
+	InstanceId string `json:"instance_id" required:"true"`
+}
+
 type RestoreToNewOptsBuilder interface {
 	ToBackupRestoreMap() (map[string]interface{}, error)
 }
 
+type RestorePITROptsBuilder interface {
+	ToBackupRestoreMap() (map[string]interface{}, error)
+}
```

Comment:
Those two interfaces have the same-named methods, so they are identical. Please rename one of the methods or both of them.